pages/news-and-events/index.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <div class="events-page">
+    <breadcrumb :breadcrumb="breadcrumb" :title="title" />
+
+    <div class="page-wrap">
+      <div class="container">
+        <h2>Events</h2>
+        <tab-nav
```

Comment:
I know this is using a slightly different layout than the `DetailTabs` component we were using for the detail pages, but the props that it pulls in are pretty much the same. Do you think it makes sense to try to use the existing component and manipulate it in this component style-wise to have it be displayed in the way you want? Just thinking in terms of reusable components.

Reply:
Yes indeed. I do want to create a generic component for all of this, but wanted to create a ticket for this to do later in.